mkdocs.yml

```diff
@@ -73,8 +73,11 @@ nav:
     - 'Building and Installing software':
       - build-install/index.md
       - 'uenv': build-install/uenv.md
-      - 'containers (podman)': build-install/containers.md
+      - 'Containers (podman)': build-install/containers.md
       - 'Python with pip': build-install/pip.md
+      - 'Applications':
```

Comment:
Seeing "Applications" in the navigation tree I would not necessarily expect to find this guide under that heading. What about using the title of the page, "Application installation guides"? Or "Application guides" like on the "Building and Installing software" index page?